### PR TITLE
Changes to cache.sh.

### PR DESCRIPTION
**Pull request by KingPachirisu**
Hi,
i fixed the issues of previous cache.sh executable.I used the getconf command to fetch information regarding caches , by doing that the programm should work for every different cache architecture.Please after checking the code ,inform me if a change must be made.I am planning no adding more to memory.sh too sooner , or later.

### CODE REVIEW

File: scripts/memory/cache.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	fi
+	let k=$(expr $k + 1 )
+done
+for s in ${strings[*]}
+do
+	if [ ${nums[$l]} -eq 0 ]
+	then 
+		echo VALUE ADV $s \" doesnt exist \"
+	else 
+		echo VALUE ADV $s \" ${nums[$l]}
```
Comment (MaanooAk) [inline scripts/memory/cache.sh]: Missing a closing `"`

File: scripts/memory/ram.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+
+totalMem=$(grep Mem /proc/meminfo | head -n 1 | cut -f 2 -d : )
+freeMem=$(grep Mem /proc/meminfo | head -n 2 | tail -n 1 | cut -f 2 -d : )
+swapMem=$(grep Mem /proc/meminfo | head -n 3 | tail -n 1 | cut -f 2 -d : )
+
+echo VALUE BAS memory_total \" $totalMem \"
+echo VALUE BAS memory_free \" freeMem \"
```
Comment (MaanooAk) [inline scripts/memory/ram.sh]: Missing a `$`

File: scripts/memory/cache.sh [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+	if [ ${nums[$l]} -eq 0 ]
+	then
+		echo VALUE BAS $s \" does not exist \"
+	else
+		numOfCaches=$( expr $numOfCaches + 1 )
+		echo VALUE BAS $s \" ${nums[$l]} \"
+	fi	
+	let l=$( expr $l + 1 )
+done
+echo VALUE BAS numOfCaches \" $numOfCaches \"
```
Comment (MaanooAk) [inline scripts/memory/cache.sh]: Use snake_case

**Comment by KingPachirisu** [comment]
i made the changes.does it work well now?

**Comment by MaanooAk** [comment]
Everything looks ok.

**Pull request merged**